packages/acceptit-Tests.package/ACCreateUserStoryTest.class/instance/testItHasCorrectClassName.st

```diff
@@ -1,3 +1,3 @@
 tests
 testItHasCorrectClassName
-	self assert: self userStoryName equals: self createdClass name.
\ No newline at end of file
+	self assert: self userStoryNameTrimmed equals: self createdClass name.
```

Comment:
You mentioned in the other PR that you changed the phrase "UserStoryName" to 'title' in order to be consistent with the naming. So this needs to be changed as well, right?